tika/server.go

```diff
@@ -36,20 +36,20 @@ import (
 // There is no need to create a Server for an already running Tika Server
 // since you can pass its URL directly to a Client.
 type Server struct {
-	jar  string
-	url  string // url is derived from port.
-	port string
-	cmd  *exec.Cmd
+	jar   string
+	url   string // url is derived from port.
+	port  string
+	cmd   *exec.Cmd
 	child *ChildOptions
```

Comment:
I think I'd prefer to add a `tmp string` field that people can set like they can with `ChildOptions`.

I'm not really keen on adding a setter (https://golang.org/doc/effective_go.html#Getters). Maybe we could just export the Tmp field? That wouldn't fit with the other arguments, though.

That way, we don't need a separate `Start` function that will be difficult to keep in sync.

Reply:
Will do. thanks.